content/boilerplates/notes/1.1.4.md

```diff
@@ -0,0 +1,42 @@
+## Behavior change
+
+- Changed the default behavior for Pilot to enable fall through routes which allows workloads to access external host using the same port as any
```

Comment:
Fallthrough route isn't really something a user needs to know about (or a envoy/istio term). Maybe just make this say that all external traffic will now be allowed, even if it is on the same port as an internal service.?

Reply:
Rephrased. PTAL.

Reply:
yes, i like that better.  That is essentially why i asked the q :)